packages/middleware-logger/src/loggerMiddleware.ts

```diff
@@ -26,8 +55,9 @@ export const loggerMiddlewareOptions: FinalizeRequestHandlerOptions & AbsoluteLo
   step: "finalizeRequest",
 };
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why do we need this?

Reply:
like why not just remove options?

Reply:
Was explained in commit description b5c87627123d0416bc5e8f7f51192635dac9a962

CodeGen passes options to getXPlugin commands, but loggerMiddleware reads logger from context. A long-term fix would need discussion on whether `getXPlugin` should work without config passed to it.

Reply:
Can we remove `options` parameter? I think it should not break the TS.

Reply:
It does break TS with:
```console
Expected 1 arguments, but got 0
```